content/doc/developer/internationalization/internationalization-and-localization.adoc

```diff
@@ -0,0 +1,301 @@
+---
+title: Internationalization and Localization
+layout: developersection
+---
+
+https://www.jenkins.io/doc/book/using/using-local-language/[How
+to view Jenkins in your language].
+
+'''''
+
+== What developers need to know/do
```

Comment:
Simpler phrasing of the heading? (optional)
```suggestion
== Developer tasks
```

Reply:
Done.